apps/mobile-wallet/src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -77,16 +68,21 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
   const isMnemonicBackedUp = useAppSelector((s) => s.wallet.isMnemonicBackedUp)
   const needsFundPasswordReminder = useAppSelector((s) => s.fundPassword.needsReminder)
 
-  const [isFundPasswordReminderModalOpen, setIsFundPasswordReminderModalOpen] = useState(false)
-  const [isBackupReminderModalOpen, setIsBackupReminderModalOpen] = useState(!isMnemonicBackedUp)
-  const [isSwitchNetworkModalOpen, setIsSwitchNetworkModalOpen] = useState(false)
   const { data: isNewWallet } = useAsyncData(getIsNewWallet)
 
+  useEffect(() => {
+    if (isNewWallet === undefined) return
+
+    if (!isMnemonicBackedUp) {
+      dispatch(openModal({ name: 'BackupReminderModal', props: { isNewWallet } }))
+    }
```

Comment:
nit: I'd merge these 2 conditions in one

```suggestion
    if (!isMnemonicBackedUp && isNewWallet !== undefined) {
      dispatch(openModal({ name: 'BackupReminderModal', props: { isNewWallet } }))
    }
```

Reply:
Done